client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-identify/index.tsx

```diff
@@ -23,8 +25,20 @@ interface Props {
 
 export const Analyzer: FC< Props > = ( { onComplete, onSkip, hideImporterListLink = false } ) => {
 	const translate = useTranslate();
+	const isEnglishLocale = useIsEnglishLocale();
 	const [ siteURL, setSiteURL ] = useState< string >( '' );
 
+	// TODO: Remove extra steps for non-English locales once we have translations.
+	const oldSubtitle = translate( 'Drop your current site address below to get started.' );
+	const newSubtitle = translate(
+		"Drop your current site address below to get started. In the next step, we'll measure your site's performance and confirm its eligibility for migration."
+	);
+	const isTranslationAvailableForNewSubtitle = hasTranslation(
+		"Drop your current site address below to get started. In the next step, we'll measure your site's performance and confirm its eligibility for migration."
+	);
+	const subtitleInUse =
```

Comment:
Nice! =)
It makes me think that at some point we could create a function on `@automattic/i18n-utils` to simplify this logic.

Reply:
Maybe I'll try to send a PR proposal this week.

Reply:
Marking it as resolved because we've decided to use the existing `useHasEnTranslation` instead